transport/dtls/SrtpClient.cpp

```diff
@@ -847,6 +847,12 @@ void SrtpClient::setRemoteKey(const srtp::AesKey& key)
     {
         _state = State::CONNECTED;
         _mode = srtp::Mode::NULL_CIPHER;
+        logger::debug("SRTP connected with null cipher", _loggableId.c_str());
+        if (_eventSink)
+        {
+            _eventSink->onDtlsStateChange(this, _state);
```

Comment:
Maybe this should be called onSrtpStateChange becasue it has to be signalled also in null cipher mode and SDES that we are connected on SRTP level.